scheduler/pkg/plugins/wasm/guest.go

```diff
@@ -1,4 +1,4 @@
-package plugin
```

Comment:
Nit: folder `scheduler` might not be necessary because this is for kube-scheduler.

Reply:
I'd like to keep the scheduler folder so that we can separate the multiple go modules in serial

```
./guest: guest wasm Go module
./scheduler: host of wasm guest. (actually, the plugin in it is the guest though)
```